_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+%% Journal of Open Research Software Latex template -- Created By Stephen Bonner and John Brennan, Durham Universtiy, UK.
+
+\documentclass{jors}
+
+%% Set the header information
+\pagestyle{fancy}
+\definecolor{mygray}{gray}{0.6}
+\renewcommand\headrule{}
+
+\usepackage{biblatex}
+\usepackage{hyperref}
+\bibliography{paper}
+
+\usepackage{graphicx}
+\usepackage{hyperref}
+
+\setlength{\parskip}{\baselineskip}%
+\setlength{\parindent}{0pt}%
+
+\begin{document}
+
+\newcommand\githublink[1]{\href{https://github.com/#1/}{\texttt{\textbf{@#1}}}}
+%% {\bf Software paper for submission to the Journal of Open Research Software} \\
+
+%% To complete this template, please replace the blue text with your own. The paper has three main sections: (1) Overview; (2) Availability; (3) Reuse potential. \\
+
+%% Please submit the completed paper to: editor.jors@ubiquitypress.com
+
+%% \rule{\textwidth}{1pt}
+
+\section*{(1) Overview}
+
+\vspace{0.5cm}
+
+\section*{Title}
+\par\bigskip
+PFHub: The Phase-Field Community Hub\\[\baselineskip]
+\url{https://pages.nist.gov/pfhub}
+
+\section*{Paper Authors}
+
+1. Wheeler, Daniel; (corresponding author) A\\
+2. Keller, Trevor; A\\
+3. DeWitt, Stephen J.; E\\
+4. Jokisaari, Andrea M.; D\\
+5. Schwen, Daniel; D\\
+6. Guyer, Jonathan E.; A\\
+7. Aagesen, Larry; D\\
+8. Heinonen, Olle G.; B\\
+9. Tonks, Michael R.; G\\
+10. Voorhees, Peter W.; C\\
+11. Warren, James A.; F
+
+\section*{Paper Author Roles and Affiliations}
+
+A. Materials Science and Engineering Division, \\
+Material Measurement Laboratory, \\
+National Institute of Standards and Technology,\\
+Gaithersburg, MD 20899 USA
+
+B. Argonne National Laboratory, \\
+Lemont, IL 60439 USA
+
+C. Department of Materials Science and Engineering, \\
+Northwestern University, \\
+Evanston, IL 60208 USA
+
+D. Fuel Modeling and Simulation Department, \\
+Idaho National Laboratory, \\
+Idaho Falls, ID 83415 USA
+
+E. Materials Science and Engineering Department, \\
+University of Michigan, \\
+Ann Arbor, MI 48109 USA
+
+F. Material Measurement Laboratory Office, \\
+Material Measurement Laboratory, \\
+National Institute of Standards and Technology,\\
+Gaithersburg, MD 20899 USA
+
+G. Department of Materials Science and Engineering, \\
+University of Florida, \\
+Gainesville, FL 32611
+
+\section*{Abstract}
+
+An online portal provides a valuable space for scientific communities
+to summarize a shared challenge, collect attempts at a solution, and
+present a quantitative comparison of past attempts in a compelling
+way. An exemplar of such a portal is
+$\mu$MAG~\cite{mumag}. The reusable PFHub
+framework leverages existing online services to build a static portal
+website that is considerably easier to deploy and maintain without
+sacrificing content or scope. The first deployment of the PFHub
+framework supports phase-field practitioners and code developers
+participating in an effort to improve quality assurance for
+phase-field codes.
+
```

Comment:
I find this abstract to be too condensed.  I would suggest a rewrite along the lines...

Scientific communities have always struggled with the challenge of efficiently and effectively sharing information.  An online portal provides a valuable space for scientific communities to summarize a shared challenge (e.g., <something>), collect attempts at a solution, and present a quantitative comparison of past attempts in a compelling way. An exemplar of such a portal is $\mu$MAG~\cite{mumag}, <muMag History Blurb>. In this work, we present a description of a generic online hub framework that may be reused and adopted by different scientific communities.  The PFHub framework leverages existing online services to build a static portal website that is considerably easier to deploy and maintain without sacrificing content or scope. The first deployment of the PFHub framework supports phase-field practitioners and code developers participating in an effort to improve quality assurance for phase-field codes. <Value-added statement, like "This deployment has enabled direct comparison of different phase field solver performance and aided in teaching of learners to phase field methodology.">

Reply:
Thanks for the feedback.

Regarding the bare-bones nature of the paper. It is a software meta-paper, which is supposed to be quite short. For example, JOSS asks for between 250 to 1000 words for its meta-papers. JORS doesn't seem to specify anything, but it has another longer paper style which asks for between 3000 and 4000 words so certainly shorter than that. Looking through the software meta-paper articles in the journal, it seem that they are about 1000 to 2000 words. With the latest changes, we are close to 1800 words which is getting quite long for this style of paper (using `detex paper.tex | wc`).

Regarding the abstract, it is actually supposed to be approximately 100 words (from the JORS TeX template). The one you have above is 130 words, which might be too long so I've knocked  it back to 109 words. Anyway, see what you think. It is very similar to what you have above.